heareval/embeddings/task_embeddings.py

```diff
@@ -101,7 +101,12 @@ def as_tensor(self, x: Union[np.ndarray, torch.Tensor]):
                 )
 
         elif self.type == TENSORFLOW:
-            NotImplementedError("TensorFlow not implemented yet")
+            # Load array as tensor onto device
+            
+            if not isinstance(x, np.ndarray):
+                x = x.numpy()
+            #x = torch.tensor(x, device=self.device)
+            x = tf.convert_to_tensor(x)
```

Comment:
@jonnor thank you for the PR. Two questions:

1) Will this be potentially slow, to move from torch GPU to numpy CPU to TF? If so, do you have a proposal how we can improve the speed?
2) How do we ensure that the TF tensor is on the desired device?

Reply:
@turian

1. my understanding of PyTorch dataloaders is that data is on CPU by default, and then you must send to GPU. If you look at the code above for the torch version that is where the move to the GPU occurs. So this should be fine from a performance perspective. 
2. TF handles that behind the scenes, we don't have a command line option to specify device so we can just rely on TF defaults.

Reply:
Hi @turian. I am in agreement with @jorshi here. The PyTorch data tensors are at (likely) at this point on the CPU, and transfer to GPU should happen after that. That said, I never intended this to be the optimally-performing solution. This is mostly a PR to get Tensorflow from not-working-at-all to basically-working, in the simplest manner possible, and without complicating things for the PyTorch code.

Reply:
@jonnor Given your experience with TF, could you suggest to us what you think would be optimally performing? We can try implementing it. What sort of speed difference could we achieve with your version versus optimal do you think?